shell/client/admin/user-details.html

```diff
@@ -285,6 +285,85 @@
     <h2>Identities</h2>
     {{> newAdminUserDetailsIdentityTable account=account }}
 
+    <h2>Status</h2>
+
+    <div style="display: flex; flex-direction: row;">
+      {{#if accountDeleting}}
```

Comment:
If you use `#with` here, then you can use `{{.}}` on the next line thus avoiding calling `accountDeleting()` twice.